client/dist/styles/ratingsReviews.css

```diff
@@ -677,4 +673,19 @@
 .r-required {
   color: red;
   font-size: small;
+}
+
+.r-full-min{
+  padding-right: 15px;
```

Comment:
I know Leslie was saying that we ought to try and use relative sizing when possible-- most of my code is still in px too, however. Something we can consider eventually refactoring if we start optimizing for mobile